app/assets/javascripts/controllers/vm_cloud/vm_cloud_resize_form_controller.js

```diff
@@ -1,36 +1,41 @@
-ManageIQ.angular.app.controller('vmCloudResizeFormController', ['$http', '$scope', 'vmCloudResizeFormId', 'miqService', function($http, $scope, vmCloudResizeFormId, miqService) {
+ManageIQ.angular.app.controller('vmCloudResizeFormController', ['$http', '$scope', 'vmCloudResizeFormId', 'objectId', 'miqService', function($http, $scope, vmCloudResizeFormId, objectId, miqService) {
   var vm = this;
 
   var init = function() {
     vm.vmCloudModel = {
       flavor_id: null,
     };
     vm.flavors = [];
-    vm.formId = vmCloudResizeFormId;
-    vm.modelCopy = angular.copy(vm.vmCloudModel);
```

Comment:
missing modelCopy for new

Reply:
See comment above from @ZitaNemeckova -- modelCopy has been moved into getResizeFormData since it has to be called after setting everything in vmCloudMOdel

Reply:
Ah, sorry about that. Usually `get*FormData` only gets called when editing, not new. 

But you're right, in your case it handles both, withdrawing my objection :).